### ISSUE TITLE
Release Twisted 9.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3860]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3860|
|Type|task|
|Created|2009-05-30 17:06:34Z|
|Branch|https://github.com/twisted/twisted/tree/releases/release-9.0.x-3860-5|

It is a moral imperative.

<details><summary>Searchable metadata</summary>

```
trac-id__3860 3860
type__task task
reporter__exarkun exarkun
priority__highest highest
milestone__Twisted_9_0 Twisted-9.0
branch__branches_releases_release_9_0_x_3860_5 branches/releases/release-9.0.x-3860-5
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1243703194000000 1243703194000000
changetime__1259187103000000 1259187103000000
version__None None
owner__ 
cc__jml cc__glyph cc__radix cc__exarkun cc__therve cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

I believe this has a fairly obvious milestone associated with it.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph commented|
|-|-|

Some work has already taken place writing news files for this release, but it's stalled.  The branch will need to be merged forward in order to incorporate the changes to trunk since it was made, including fixes for a couple of regressions.

Since release branch names violate the standard we use for other branch names, this didn't show up in the "branch" field.  I am not sure if I should update it, so instead here's a comment that refers to it: [branches/releases/release-9.0.x-3860 /branches/releases/release-9.0.x-3860](branches/releases/release-9.0.x-3860 /branches/releases/release-9.0.x-3860).

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @jml|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_5"></a>|@radix removed owner|
|-|-|

READY FOR REVIEW

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

It looks okay to me.  I certainly didn't manage to pay attention long enough to read every single news item, though.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @radix|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_8"></a>|@radix set status to `closed`|
|-|-|

(In [27607]) Merge releases/release-9.0.x-3860-5

Author: radix
Reviewer: exarkun
Fixes: [#3860](https://github.com/twisted/twisted/issues/3860)

Update all the version numbers and add NEWS entries for the 9.0 release.
In addition, LoopingCall.withCount's "@since" property was changed to
say 9.0 instead of 9.1.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
